content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -2,264 +2,192 @@
-  - `Deferred`: This value means that the requested resize cannot be granted at
-    this time, and the node will keep retrying. The resize may be granted when
-    other pods are removed and free up node resources.
-  - `Infeasible`: is a signal that the node cannot accommodate the requested
-    resize. This can happen if the requested resize exceeds the maximum
-    resources the node can ever allocate for a pod.
-  - `""`: An empty or unset value indicates that the last resize completed. This should only be the
-    case if the resources in the container spec match the resources in the container status.
-
-If a node has pods with an incomplete resize, the scheduler will compute the pod requests from the
-maximum of a container's desired resource requests, and it's actual requests reported in the status.
+This page explains how to change the CPU and memory resource requests and limits assigned to a container *without recreating the Pod*.
 
+Traditionally, changing a Pod's resource requirements necessitated deleting the existing Pod and creating a replacement, often managed by a [workload controller](/docs/concepts/workloads/controllers/). In-place resizing allows containers within a running Pod to be allocated more or less CPU/memory, potentially avoiding application disruption.
+
+**Key Concepts:**
+
+* **Desired Resources:** A container's `spec.containers[*].resources` represent the *desired* resources for the container, and are mutable for CPU and memory.
+* **Actual Resources:** The `status.containerStatuses[*].resources` field reflects the resources *currently applied* for a running container. For containers that haven't started or were restarted, it reflects the resources allocated upon their next start.
+* **Triggering a Resize:** You request a resize by updating the desired `requests` and `limits` in the Pod's specification. This is typically done using `kubectl patch`, `kubectl apply`, or `kubectl edit` targeting the Pod's `resize` subresource. When the desired resources don't match the allocated resources, the Kubelet will attempt to resize the container.
+* **Allocated Resources (Advanced):** The `status.containerStatuses[*].allocatedResources` field tracks resource values confirmed by the Kubelet, primarily used for internal scheduling logic. For most monitoring and validation purposes, focus on `status.containerStatuses[*].resources`.
+
+If a node has pods with a pending or incomplete resize (see [Pod Resize Status](#pod-resize-status) below), the scheduler uses the *maximum* of a container's desired requests, allocated requests, and actual requests from the status when making scheduling decisions.
 
 ## {{% heading "prerequisites" %}}
 
 
 {{< include "task-tutorial-prereqs.md" >}} {{< version-check >}}
 
-The `InPlacePodVerticalScaling` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) must be enabled
-for your control plane and for all nodes in your cluster.
```

Comment:
This is still true. I wouldn't drop it yet.

Reply:
Ack. I wasn't sure if the feature gate header was sufficient to cover this.